app/src/main/res/values/strings.xml

```diff
@@ -191,12 +191,21 @@
     <!-- Broken Site Activity -->
     <string name="brokenSiteReportBrokenSiteMenuItem">Report Broken Site</string>
     <string name="brokenSiteHeading">Report a Broken Site</string>
-    <string name="brokenSiteModalDescription">Broken site reporting is completely anonymous and helps us improve the app!</string>
-    <string name="brokenSiteUrlHint">Enter URL</string>
-    <string name="brokenSiteDomainLabel">Domain of broken site:</string>
-    <string name="brokenSiteMessageHint">Which content or functionality is breaking?</string>
-    <string name="brokenSiteSubmitButton">Submit Feedback</string>
     <string name="brokenSiteSubmitted">Thank you! Feedback submitted</string>
+    <string name="brokenSitesCategoriesTitle">Describe what happened:</string>
+    <string name="brokenSitesCategoriesHint">Describe what happened</string>
+    <string name="brokenSiteSubmitButton">Send Report</string>
+    <string name="brokenSiteDescription">Submitting an anonymous broken site report helps us debug these issues and improve the app.</string>
+
+    <string name="brokenSiteCategoryImages">Images didn\'t load</string>
+    <string name="brokenSiteCategoryPaywall">The site asked me to disable</string>
```

Comment:
Apologies if you answered this somewhere already; i can't find my earlier question on it.

Still don't understand this copy. Is this right?

Reply:
It is yes, approved here https://app.asana.com/0/inbox/1125189844075764/1165569833751685/1166866723173161

Reply:
Discussed offline; this was deliberate. Don't think it makes sense, but not a blocker.